\documentclass{article}

\usepackage{bm}

\title{Overview, Design concepts, and Details (ODD) protocol}

\begin{document}

\maketitle

\section{Purpose}

The model aims at simulating the evolution of a gene regulatory network in a population-genetic context. 

\section{Entities, state variales, and scales}

The model is individual-based. Individuals are characterized by their genotype (their gene network) and their phenotype (the expression level of each network gene when the network is at equilibrium. 

Generations are non-overlapping, and at a given generation, the state of the model is defined by the $N$ individuals of the population. The model defines explicilty two time scales. 

At the network time scale, the kinetics of an individual's gene network is calculated, resulting in the expression phenotype for this individual. The network kinetics is a discrete process, featured by 16 cycles of gene expression update. Each time step roughly corresponds to the life time of a gene product, i.e. it correpsonds in real life to several hours.

Evolution happens at the population time scale. The population scale is also discrete, with non-overlapping generations. A generation consists in generating new individuals from parents, accounting for selection (differential reproduction probability depending on the phenotype) and mutation (random change in the genotype). Genetic drift arise spontaneously as the consequence of sampling genotypes in a finite population size. The time scale of generations can span weeks to years according to the species. 

\paragraph{Agents and Individuals}

Each individual has a fitness $w$, which stands for its propensity to reproduce. 

\paragraph{Spatial units}

The model does not consider spatial dimensions, mating probability only depends on fitness. 

\paragraph{Environment}

The state of the environment at each generation $g$ is represented by a single variable $0 < e_g < 1$. The environment variable is sampled in a uniform distribution every generation, and is uncorrelated across generations (white noise process: $\cov (e_g, e_{g+1}) = 0$). 

The environment has two distinct (although totally correlated) effects in our model: (i) it affects the expression level of a "signal" gene in the network (this gene is expressed at level $e_g$ throughout the whole network kinetics), and (ii) it affects the fitness optimum of a subset of the gene networks (the "plastic genes"). In practice, the optimal expression level for half of the plastic genes is $e_g$, and $1-e_g$ for the other half. 

The model does not include within-generation environmental stochasticity, so that the genotype-phenotype map is deterministic (no microenvironmental variance). 

\section{Process overview and scheduling}

\section{Design concepts}

\subsection{Basic principles}

\subsection{Emergence}

\subsection{Adaptation and objectives}

Adaptation is driven by selection. Each individual $j$ is attributed a fitness score $w_j$, which is proportional to the probability to be picked as a parent. The model implements "soft selection", i.e. fitnesses are relative and the population size does not depend on the mean fitness value. 

The fitness function has two components: $w = w_I \cdot w_S$. The first component $w_I$ corresponds to the penalty for unstable networks, $w_I = \prod_{i=1}^n \exp(-s^\prime V_i)$, where $s^\prime$ is the selection coefficient against unstability, and $V_i$ the variance in gene $i$ expression over the four last cycles of the network kinetics. The coefficient $s^\prime$ was set to a large value ($s^\prime=46000$) to make unstable networks virtually unviable; this (disputable) assumption being generally made in the litterature. The second component $w_S = \prod_{i=1}^n exp(-s_i(p_i-\theta_i)^2)$ is the stabilizing selection component, providing a reproductive advantage to individuals close to the optimal phenotype $\theta$ for all genes. The selection coefficient $s_i$ scales with the inverse variance of this Gaussian function, it was set to 10 for "selected" genes (which ensures a gentle, but substantial, stabilizing selection), and 0 for non-selected genes. 

\subsection{Learning}

There is no form of learning in the model beyond genetic adaptation to stable and fluctuating environments. 

\subsection{Prediction}

The model aims at describing evolution by natural selection, which is by defintion unable to account for future or potential benefits. The environment is not predictable (the environmental variable $e_g$ is drawn in an independent uniform distribtuion every generation). Yet, past adaptation to a fluctuating environment could lead to the evolution of phenotypic plasticity, which is an indirect way to adapt to unpredictable fluctuations. 

\subsection{Sensing}

The value of the environmental variable $e_g$ at each generation $g$ is available to the networks \emph{via} a sensor gene, which is expressed at level $e_g$ during the whole network dynamics. The network thus has a way to "sense" the environment and adapts its response to it (to evolve phenotypic plasticity). 

\subsection{Interaction}

\paragraph{Between individuals} There is no interaction among individuals, beyond the fact that the population size is fixed and deterministic (soft selection). Individuals reproduce under a random mating regime, and there is no population structure. The probability of generating full sibs is the same as the probability of drawing twice the same pair of parents, and the probability of selfing is the probability of drawing twice the same parent. 

\paragraph{Between genes} The model features regulatory interactions between genes. The regulation pattern is determined by the genotype of each individual, coded as an interaction matrix. Interactions are asymmetric: if gene A regulates gene B, gene B may not necessarily regulate gene A (although the possibility exists). Interactions can be positive (up-regulation) or negative (down-regulation). A non-zero interaction component in the matrix does not necessarily imply a functional interaction, for instance if the regulatory gene is not expressed. 

Gene interactions are additive and there is no third-order interactions (e.g. between triplets of genes) at a single network time step. Complex and non-linear interactions arise due to (i) the non-linear (sigmoid) scaling function that scales the regulated expression to the (0,1) interval, and (ii) the repetition of regulation cycles, whih allows for any kind of feedback and feedforward loops depending on the network topology. 

Interactions in the gene network are expected to lead to a certain degree of genetic interactions (epistasis), in which the effect of a genetic change on the (expression) phenotype depends on the genetic background.

\subsection{Stochasticity}

\paragraph{Genetic drift} Reproduction involves a sampling protocol. For each offspring, two individuals are sampled from the parental population with a probability that is proportional to their fitness. The contribution of an individual to the next generation is thus a random number drawn from a Poisson distrubution. 

\paragraph{Recombination} Diploid individuals generate haploid gametes by sampling loci from the haplotypes they recieved from their parents. Recombination is free, i.e.\ alleles are sampled independently. 

\paragraph{Mutations} Mutations are random changes in the value of alleles (\emph{cis}-regulatory sites). Mutations occur at a givent rate $\mu$ per haplotype and per generation, during gametogenesis. The position of the mutation in the genotype ($\mathrm W$) matrix is random and uniform. A mutation consists in adding a Gaussian random deviate (of standard deviation $\sigma_m$) to the original allelic value. 

\paragraph{Intial genotype} The initial genotype matrtix is initialized with very small interaction strengths (drawn in a Gaussian of mean 0 and standard deviation 0.001), that are different across replicates (but identical across individuals). These deviates are so small that they are not expected to affect the simulation results, and this is unlikely to bring a significant source of stochasticity to the simulation.

\paragraph{Environment} The generation-specific environment is summarized by an index $0 < e_g < 1$, which is drawn from a uniform distribution every generation (white noise process).

\paragraph{Fitness optima} The model considers two sets of genes that are under direct selection. For "plastic" genes, the optimum follows the environental index and thus changes every generation. For "stable" genes, the optimum is drawn at the beginning of the simulation in a uniform (0,1) distribution, and does not change through time. For a subset of genes, the optima change once (at the "domestication") event, old optima being replaced by new ones drawn in the same distribution. The value of fitness optima is independent across simulation replicates. 

\subsection{Collectives}

Non applicable.

\subsection{Observation}

\paragraph{Genotype and phenotype} Each individual $j$ is featured by its genotype (a $n \times n$ regulation matrix $\bm{\mathrm W_j}$), its expression phenotype $p_{ij}$ at every gene $i$, and the variance $V_{ij}$ in gene expression during the four time steps preceding the end of the network kinetics for each gene $i$. A fitness score $w_j$ is also calculated from the phenotype. 

\paragraph{Summary statistics} The simulation software reports summary statistics on a regular basis (every generation if necessary). Summary statistics include: $\bar p_i$, the population average gene expression at gene $i$; $\bar V_i$, the population average expression unstability at gene $i$, and $\bar w$, the average fitness in the population. In addition, the respective population variances $V_{p_i}$, $V_{V_i}$, and $V_w$ are also provided, as well as the phenotypic covariances between the expression of each pair of genes ($i, i^\prime$), $V_{p_{i,i^\prime}}$. As the model does not include within-generation environmental noise, the phenotypic (co)variances also represent genetic (co)variances. Finally, the program provides the population mean $\bar W_{i,i^\prime}$ and the population variance $V_{W_{i,i^\prime}}$ of each genetic component. 

\paragraph{Average network} To simplify the computational burden of calculating summary statistics, the evolutionary dynamics of gene networks was studied based on the average genotype $\bar{\bm{\mathrm W}}$ at each generation. This average genotype was analyzed as a "standard" network; in most cases, the difference between e.g. gene expressions calculated from the average network vs.\ average gene expressions in the population were very small, supporting the approximation. In practice, the average network was used to (i) estimate phenotypic plasticity (slope of a linear regression between 11 regularly spaced environmental values between 0 and 1 vs. the expression phenotype of the considered gene), and (ii) estimate network topology. 

\paragraph{Network topology} As the genetic model is quantitative, there was no true "zeros" in the $\bm{\mathrm W}$ matrix, requesting to rely on some thresholds to determine whether or not an interaction exists. In practice, we replaced one by one all interactions by real zeros, and computed gene expression with ($\bm{\mathrm P}$) and without ($\bm{\mathrm P_{0_{i,i^\prime}}}$) the interaction between genes $i$ and $i^\prime$. If the euclidian distance between both gene expression vectors $\delta_{i,i^\prime} = \sqrt{(\bm{\mathrm  P_{0_{i,i^\prime}}} - \bm{\mathrm P}) \cdot (\bm{\mathrm  P_{0_{i,i^\prime}}} - \bm{\mathrm P})}$ was larger than a threshold $\tau$, then the interaction between $i$ and $i^\prime$ was included in the model. + modules and other things calculated from topology. 

\section{Initialization and parameterization}

Simulations were initialized with "empty" genetic networks $\bm{\mathrm W}$. In practice, because the simulation software gives a special meaning to "true" zeros in the genotype, a starting network was determined by drawing $n \times n$ random numbers in a Gaussian of mean 0 and standard deviation 0.0001, which has in practice no effect on gene regulation. This starting network was then duplicated in all individuals. 

The fixed optima were randomly drawn for each simulation in uniform (0,1) distributions. 

\section{Input data}

\paragraph{Parameter set}

\begin{center}
\begin{tabular}{p{4cm}clc}
Parameter & Symbol & Code & Default value \\ \hline
Total number of loci & $n$ & \texttt{GENET\_NBLOC} & 25 \\
Recombination rate & $r$ & \texttt{GENET\_RECRATE} & 0.5 \\
Mutation rate (per haplotype per generation) & $\mu$ & \texttt{GENET\_MUTRATES} & ? \\
Standard deviation of mutational effects & $\sigma_m$ & \texttt{GENET\_MUTSD} & 0.1 \\
Ploidy level & &  \texttt{GENET\_PLOIDY} & 2 \\
Probability of self-fertilization & & \texttt{GENET\_SELFING} & 0 \\
Probability of clonal reprodiction & & \texttt{GENET\_CLONAL} & 0 \\
Number of regulation time steps & & \texttt{DEV\_TIMESTEPS} & 16 \\
Number of time steps on which stability is computed & & \texttt{DEV\_CALCSTEPS} & 4 \\
Constitutive expression & $a$ & \texttt{INIT\_BASAL} & 0.2 \\
Initial distribution of allelic values (mean and sd)& & \texttt{INIT\_ALLELES} & 0; 0.0001 \\
Strength of stabilizing selection & $s$ & \texttt{FITNESS\_STRENGTH} & 10 \\
Strength of selection on network stability & $s^\prime$ & \texttt{FITNESS\_STABSTR} & 46000 \\ \hline
Population size & $N$ & \texttt{INIT\_PSIZE} & variable \\
Selection optimum & $\theta$ & \texttt{FITNESS\_OPTIMUM} & variable \\ 
\end{tabular}
\end{center}

\paragraph{Domestication scenario} 



\section{Submodels}





\end{document}
